evol/population.py

```diff
@@ -327,6 +340,36 @@ def _update_documented_best(self):
             self.documented_best = copy(current_best)
 
 
+class Contest(object):
```

Comment:
In python3 we no longer need to inherit from `object`; i.e. `class Contest:` should suffice.